qtpy/enums_compat.py

```diff
@@ -37,3 +37,24 @@ def promote_enums(module):
                     continue
                 for name, value in attrib.__members__.items():
                     setattr(klass, name, value)
+
+if PYQT5:
+    def demote_enums(module):
+        """
+        Search unscoped enums in the given module and allow scoped access.
+
+        Special for PyQt5==5.9.*
+        """
+        class LookUp:
+            def __init__(self, lookup_location):
+                self._ll = lookup_location
+
+            def __getattr__(self, what):
+                return getattr(self._ll, what)
+
+        for class_name in dir(module):
+            if class_name == 'Qt' or (class_name[0] == 'Q' and class_name[1].isupper()):
+                klass = getattr(module, class_name)
+                for attrib_name in dir(klass):
+                    if attrib_name[0].isupper() and type(getattr(klass, attrib_name)).__name__ == 'enum''type':
```

Comment:
```suggestion
                    if attrib_name[0].isupper() and type(getattr(klass, attrib_name)).__name__ == 'enumtype':
```

Looks like maybe an issue with Black's older string parsing?

Reply:
I've split the string just to avoid a typo reported by the IDE.

I know that the part looks ugly, but I can't think of another way of checking the type against `sip.enumtype`.

Reply:
Could you silence that with a `# noqa` comment or similar? Because otherwise it looks like a typo from my *eye*-DEs (:laughing: ). Not sure its worth this ugly hack just because one particular IDE has a false positive...